.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,15 +3,17 @@
   WIP: work in progress
   🐛    Fix a bug.
   ✨    Introduce new features.
-  ♻️     Refactor code.
+  ♻️    Refactor code.
   🚑️    Critical hotfix.
-  ⚗️     Perform experiments.
-  ⬆️     Upgrade dependencies.
+  🎨    Enhance existing feature.
```

Comment:
Please do not add 🎨 and with a different meaning than in  https://gitmoji.dev/. 

The idea is to keep a few  here that cover the most common scenarios and "we use well". Then if there is something more concrete we then use gitmoji.dev

Reply:
I will add the description from gitmoji.dev :) thanks

Reply:
@odeimaiz @pcrespov please discuss this separately, I will remove it again to get the PR through.

Reply:
not worth a discussion ... if your or @odeimaiz needs that icon, just add it :-)